docs/deployment/docker.rst

```diff
@@ -10,3 +10,12 @@ Then use Docker to build the image:
 .. code-block:: bash
 
  $ docker build -t mystorefront .
+
+
+Then you can run saleor container with the minimal settings:
+
+.. code-block:: bash
+
+ $ docker run -e SECRET_KEY=<SECRET_KEY> -e DATABASE_URL=<DATABASE_URL> -p 8000:8000 saleor
+
+For more environment variable settings, please refer to `Configuration <https://docs.getsaleor.com/en/latest/gettingstarted/configuration.html>`_
```

Comment:
You should use a `:ref:...` here instead of a hard-coded link.

Also, wouldn't it sound better by swapping the sentence? Like this:
```suggestion
Please refer to the `configuration guide <https://docs.getsaleor.com/en/latest/gettingstarted/configuration.html>`_ for more environment variables.
```

Reply:
Thanks! The hard-coded link doesn't feel right and I just forget ```:ref:``` can be used here.